spec/i18n_spec.rb

```diff
@@ -43,6 +45,7 @@
 
         (@i18n.locales - [@i18n.base_locale]).each do |other_locale|
           node = @i18n.data[other_locale].first.children[key]
+          next if node.nil?
```

Comment:
was failing here previously bc would still check for `state_file.questions.ny_sales_use_tax.edit.use_tax_method_manual_html`, I'm a little iffy if this logic is right but it did seem to fix the test

Reply:
I ran into a similar issue and added some [minimal html ](https://github.com/codeforamerica/vita-min/blob/main/config/locales/es.yml#L1805)to the translation string. Once we have spanish translations, I imagine that we'll want to make sure we have a spanish translation for every english string. That feels like a different test to me than this one which checks whether the html matches. I feel comfortable with this edit.